hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/context/properties/SemanticVersions.java

```diff
@@ -57,33 +59,76 @@ public SerializableSemVers deployedSoftwareVersion() {
 
     private void ensureLoaded() {
         if (knownActive.get() == null) {
-            final var deployed = fromResource(VERSION_INFO_RESOURCE, HAPI_VERSION_KEY, HEDERA_VERSION_KEY);
+            final var deployed = fromResource(
+                    VERSION_INFO_RESOURCE,
+                    BOOTSTRAP_PROPERTIES_RESOURCE,
+                    HAPI_VERSION_KEY,
+                    HEDERA_VERSION_KEY,
+                    HEDERA_CONFIG_VERSION_KEY);
             knownActive.set(deployed);
             knownSerializable.set(new SerializableSemVers(deployed.protoSemVer(), deployed.hederaSemVer()));
         }
     }
 
     @NonNull
-    static ActiveVersions fromResource(final String propertiesFile, final String protoKey, final String servicesKey) {
-        try (final var in = SemanticVersions.class.getClassLoader().getResourceAsStream(propertiesFile)) {
+    static ActiveVersions fromResource(
+            final String propertiesFile,
+            final String bootstrapPropertiesFile,
+            final String protoKey,
+            final String servicesKey,
+            final String configKey) {
+        try (final var in = SemanticVersions.class.getClassLoader().getResourceAsStream(propertiesFile);
+                final var bootstrapIn =
```

Comment:
Instead of this, we should just construct,
```
final var bootstrapProperties = new BootstrapProperties(false);
```
since this is the way to automatically pick up the _data/config/bootstrap.properties_ from outside the JAR.

Reply:
Thanks Michael ! Done